pilz_teleoperation/src/pilz_teleoperation/teleoperation_driver.py

```diff
@@ -78,8 +78,8 @@ def __init__(self, window):
 
     def __ros_init(self):
         self._hz = rospy.Rate(20)
-        self._sv_settings = rospy.Service("set_teleop_settings", SetTeleopSettings, self.set_teleop_settings)
-        self._sub_twist = rospy.Subscriber("teleop_twist", Twist, self.set_twist_command, queue_size=1)
+        self._sv_settings = rospy.Service("/teleoperation/set_settings", SetTeleopSettings, self.set_teleop_settings)
```

Comment:
maybe prepend `rospy.get_name()` instead of hard-coding `/teleoperation/` for easier remapping?